services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -83,7 +89,7 @@ export default class ApplyPage extends React.Component<Props> {
               phone: '',
               email: '',
               confirmEmail: '',
-              commissionIds: [] as string[],
+              commissionIds: [commissionID] as string[],
```

Comment:
I think this should change to `commissionID ? [commissionID] : []`, and then the `as string[]` can be taken out.